frontend/src/pages/pipelines/global/modelCustomization/trainingHardwareSection/TrainingHardwareProfileFormSection.tsx

```diff
@@ -75,15 +79,28 @@ const TrainingHardwareProfileFormSection: React.FC<TrainingHardwareProfileFormSe
             <Popover
               bodyContent={
                 <>
+                  <Content component={ContentVariants.p}>
+                    This list includes only hardware profiles that have GPU defined.
```

Comment:
One small question - In the doc, it is mentioned to to use `This list includes only hardware profiles with defined GPUs.`. Is this needs to be updated?